src/Utils/Create/Comments.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Console\Utils\Create\Nodes.
+ */
+
+namespace Drupal\Console\Utils\Create;
+
+use Drupal\Core\Entity\EntityManagerInterface;
+use Drupal\Core\Datetime\DateFormatterInterface;
+use Drupal\Core\Language\LanguageInterface;
+use Drupal\node\Entity\Node;
+use Drupal\comment\Entity\Comment;
+
+/**
+ * Class Nodes
+ * @package Drupal\Console\Utils
+ */
+class Comments extends Base
+{
+    /* @var array */
+    protected $bundles = [];
+
+    /**
+     * Comments constructor.
+     *
+     * @param EntityManagerInterface $entityManager
```

Comment:
If you decided to apply my other PR, we need to update this and use the entityTypeManager,  I can do it once you decide of the other PR will be merged or not.